### ISSUE TITLE
Productionize Documentation

### ISSUE THREAD

**Issue by TheToymaker93** [WMTaylor3/Arduino-CarAccessoryKit#4]
Revise documentation to make it more concise, or at least break it down into smaller chunks. Right now it's hard to parse.

**Comment by lbg-oju** [comment]
Hi, are you looking for contributors? Would you mind if I got assigned this issue?

**Comment by TheToymaker93** [comment]
> Hi, are you looking for contributors? Would you mind if I got assigned this issue?

Hi, not looking for contributors at this stage sorry. Still needing to flesh out more of what this project actually is before start involving other people. Currently it's all just in my head anyway.

Thanks for your interest!
